python/lsst/ap/association/utils.py

```diff
@@ -329,3 +330,15 @@ def ssObjectID_to_objID(ssObjectID):
 
     objID = ''.join([chr((ssObjectID >> (8 * i)) % 256) for i in reversed(range(0, 7))])
     return objID, ssObjectID >> (8 * 7) % 256
+
+
+def makeEmptyForcedSourceTable(schema):
+    """Return a dataframe with the correct columns for diaForcedSources table.
+
+    Returns
+    -------
+    diaForcedSources : `pandas.DataFrame`
+        Empty dataframe.
+    """
+    diaForcedSources = convertTableToSdmSchema(schema, pd.DataFrame(), tableName="DiaForcedSource")
+    return diaForcedSources
```

Comment:
With the other changes we have made, this function is no longer used anywhere and can be deleted. Make sure to remove it from `__all__` above, too